jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiControllerTest.java

```diff
@@ -297,9 +295,8 @@ void create_requestTenantIdDoesNotMatchPayloadTenantId_tenantIdMismatchException
     @Test
     @Transactional
     void create_requestTenantIdIsPresent_payloadTenantIdIsNotPresent_tenantIdIsSavedWithResource() throws JsonProcessingException {
-        String payload = "{\n" +
-                "            \"" + PROFILE_ID_FIELD_NAME + "\": \"" + 1 + "\"\n" +
-                "        }";
+
+        String payload = PayloadCreator.createPayload(Map.of(PROFILE_ID_FIELD_NAME, "1"));
```

Comment:
nit: the `"1"` here can be passed in as it's original type `1` since the `createPayload` map value arg is now of type `Serializable`.
Same goes for the other integer/long payload values.

Reply:
I've converted all of the payload values to integers/longs